src/index.ts

```diff
@@ -24,10 +22,10 @@ export class DatabaseError extends Error {
 
 type Types = Record<string, string>
 
-export interface ExecutedQuery {
+export interface ExecutedQuery<T> {
```

Comment:
For backwards compatibility, maybe we want to default to `Row` here?

```suggestion
export interface ExecutedQuery<T = Row> {
```